service.subtitles.supersubtitles/service.py

```diff
@@ -0,0 +1,577 @@
+__temp__ = xbmc.translatePath(os.path.join(__profile__, 'temp', ''))
+
+#sys.path.append (__resource__)
+
+BASE_URL = 'http://www.feliratok.info/index.php'
+
+TAGS = [
+    'WEB-DL',
+    'PROPER',
+    'REPACK'
+]
+
+QUALITIES = [
+    'HDTV',
+    '720p',
+    '1080p',
+    'DVDRip',
+    'BRRip',
+    'BDRip',
+    'WEB',
+    'WEBRip'
+]
+
+RELEASERS = [
+    '2HD',
+    'AFG',
+    'ASAP',
+    'BiA',
+    'DIMENSION',
+    'EVOLVE',
+    'FoV',
+    'FQM',
+    'IMMERSE',
+    'KiNGS',
+    'LOL',
+    'REMARKABLE',
+    'ORENJI',
+    'TLA',
+
+    '0SEC',
+    'FLEET',
+    'KILLERS',
+
+    'AVS',
+    'BATV'
+    'SVA',
+
+    'TBS'
+]
+
+HEADERS = {'User-Agent': 'xbmc subtitle plugin'}
+
+ARCHIVE_EXTENSIONS = [
+    '.zip',
+    '.rar',
+    '.cbr'
+]
+
+LANGUAGES = {
+    "albán": "Albanian",
+    "arab": "Arabic",
+    "bolgár": "Bulgarian",
+    "kínai": "Chinese",
+    "horvát": "Croatian",
+    "cseh": "Czech",
+    "dán": "Danish",
+    "holland": "Dutch",
+    "angol": "English",
+    "észt": "Estonian",
+    "finn": "Finnish",
+    "francia": "French",
+    "német": "German",
+    "görög": "Greek",
+    "héber": "Hebrew",
+    "hindi": "Hindi",
+    "magyar": "Hungarian",
+    "olasz": "Italian",
+    "japán": "Japanese",
+    "koreai": "Korean",
+    "lett": "Latvian",
+    "litván": "Lithuanian",
+    "macedón": "Macedonian",
+    "norvég": "Norwegian",
+    "lengyel": "Polish",
+    "portugál": "Portuguese",
+    "román": "Romanian",
+    "orosz": "Russian",
+    "szerb": "Serbian",
+    "szlovák": "Slovak",
+    "szlovén": "Slovenian",
+    "spanyol": "Spanish",
+    "svéd": "Swedish",
+    "török": "Turkish",
+}
+
+EPISODE_REGEXP = re.compile(r'S?(?P<season>\d+)([x_-]|\.)*E?(?P<episode>\d+)', re.IGNORECASE)
+
+def recreate_dir(path):
+    if xbmcvfs.exists(path):
+        try:
+            fse = sys.getfilesystemencoding()
+            if fse:
+                debuglog("Remove %s directory with file system encoding: %s" % (path, fse))
+                shutil.rmtree(py2_encode(path, fse), ignore_errors=True)
+            else:
+                debuglog("Remove %s directory with out file system encoding" % path)
+                shutil.rmtree(path, ignore_errors=True)
+        except Exception as e:
+            errorlog("Exception while delete %s: %s" % (path, e.message))
+
+    if not xbmcvfs.exists(path):
+        debuglog("Create %s directory" % path)
+        xbmcvfs.mkdirs(path)
+
+def normalize_string(str):
+    return py2_encode(unicodedata.normalize('NFKD', py2_decode(py2_encode(str, 'utf-8'))), 'ascii'
+                      #, 'ignore'
+    )
+
+
+def lang_hun2eng(hunlang):
+    return LANGUAGES[py2_encode(hunlang, "utf-8").lower()]
+
+
+def log(msg, level):
+    xbmc.log(py2_encode((u"### [%s] - %s" % (__scriptid__, msg)), 'utf-8'), level=level)
+
+
+def infolog(msg):
+    log(msg, xbmc.LOGNOTICE)
+
+
+def errorlog(msg):
+    log(msg, xbmc.LOGERROR)
```

Comment:
Not allowed. From the rules: _addons should use the xbmc.log() method to write to the logfile and use the debug logging level only. print statements should not be used._